src/components/views/dialogs/LazyLoadingDisabledDialog.tsx

```diff
@@ -19,8 +19,13 @@ import React from 'react';
 import QuestionDialog from './QuestionDialog';
 import { _t } from '../../../languageHandler';
 import SdkConfig from '../../../SdkConfig';
+import { IDialogProps } from "./IDialogProps";
 
-export default (props) => {
+interface IProps extends IDialogProps {
+    host: string;
+}
+
+const LazyLoadingDisabledDialog: React.FC<IProps> = (props: IProps) => {
```

Comment:
specifying IProps here twice is redundant, given that it doesn't take `children` (which FC implies) I would suggest `props: IProps`